.editorconfig

```diff
@@ -0,0 +1,9 @@
+root = true
```

Comment:
While I agree with all of the decisions here, I'm not sure I understand why this has been committed.